security/certificates/api-server.adoc

```diff
@@ -5,9 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The default API server certificate is issued by an internal {product-title}
-cluster CA. Clients outside of the cluster will not be able to verify the
-API server's certificate by default. This certificate can be replaced
-by one that is issued by a CA that clients trust.
+The default API server certificate is issued by an internal {product-title} cluster CA. Clients outside of the cluster cannot verify the API server's certificate by default. You can add a certificate, which is issued by a CA that clients trust, for the domains specified in your certificate to use.
```

Comment:
would add that you cannot replace the existing cert for that domain, only add your own with another domain.

Also maybe link to the section where user certs are discussed?